sqlmesh/core/config/root.py

```diff
@@ -100,6 +104,7 @@ class Config(BaseConfig):
     env_vars: t.Dict[str, str] = {}
     username: str = ""
     physical_schema_override: t.Dict[str, str] = {}
+    physical_schema_mapping: t.Dict[re.Pattern, str] = {}
```

Comment:
Instead of having both properties can we just automatically convert `physical_schema_override` into `physical_schema_mapping` by wrapping every key into `^` / `$`? This way we can just always pass the mapping and avoid extra complexity with resolved.

Reply:
I guess resolver is kind of helpful, but we can still get rid of an unnecessary field.

Reply:
Yes, that's a much better idea. I've implemented it